x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/tabs/index.tsx

```diff
@@ -145,53 +135,58 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
      */
     return (
       <>
-        <HideShowContainer
-          $isVisible={TimelineTabs.query === activeTimelineTab}
-          data-test-subj={`timeline-tab-content-${TimelineTabs.query}`}
+        <OnDemandRenderer
```

Comment:
Only load the components when you click on the tab, but it remains rendered after first load